src/datajudge/db_access.py

```diff
@@ -902,3 +902,85 @@ def get_column_array_agg(
             for t in result
         ]
     return result, selections
+
+
+def get_ks_2sample(engine: sa.engine.Engine, table1: tuple, table2: tuple):
+    """
+    Runs the query for the two-sample Kolmogorov-Smirnov test and returns the test statistic d.
+    """
+    table1_selection, col1 = table1
+    table2_selection, col2 = table2
+
+    if is_mssql(engine):
+        table1_selection = str(table1_selection).replace(
+            '"', ""
+        )  # tempdb.dbo.int_table
```

Comment:
Comments are different, accident?

Reply:
Intention was to show what the replacement should achieve, is now somewhat cleaner.